books/clients/livelibru.py

```diff
@@ -0,0 +1,89 @@
+import requests
+import urllib
+import json
+import math
+
+REQUIRED_HEADERS = {
+    "Host": "www.livelib.ru",
+    "Accept-Language": "en-US;q=1.0, ru-US;q=0.9, be-US;q=0.8",
+    "Accept": "*/*",
+    "Connection": "keep-alive",
+    "Accept-Encoding": "gzip;q=1.0, compress;q=0.5",
+    "User-Agent": "LiveLib/3.24.5 (ru.livelib.LiveLib; build:3.24.5; iOS 16.5.1) Alamofire/3.24.5"
+}
+
+
+def _generate_session_guid():
+    init_session_response = requests.get(
+        url="https://www.livelib.ru/apiapp/v2.0/appsessions/me?andyll=and7mpp4ss&fields=guid",
+        headers=REQUIRED_HEADERS
+    )
+    if init_session_response.status_code > 299:
+        raise RuntimeError(f"Filed to start new livelib session, {init_session_response.text}, " +
+                           f"response status: {init_session_response.status_code}")
+    guid = json.loads(init_session_response.text).get("data").get("guid")
+    return guid
+
+
+def _get_another_page(query_text, session_token=None, start_from=1, result_count=100):
+    if session_token:
+        stoken = session_token
+    else:
+        stoken = _generate_session_guid()
+    query = urllib.parse.quote(query_text.lower())
+    search_response = requests.get(
+        url="https://www.livelib.ru/apiapp/v2.0/search/books?andyll=and7mpp4ss&fields=author_id%2C" +
+            "author_name%2Cavg_mark%2Cid%2Ccount_reviews%2Cis_work%2Cname%2Cpic_200%2Cshare_url%28" +
+            "share_url%29%2Cuser_book_partial%28book_read%2Crating%29&"
+            f"&session_id=&count={result_count}&start={start_from}&q={query}&app_session_guid={stoken}",
```

Comment:
Let's use urlencode to improve readability of this string and make sure that we escape query:

```
urlencode({
  'andyll': 'and7mpp4ss',
  'fields': 'author_id,author_name,avg_mark,id,count_reviews,is_work,name,pic_200,share_url(share_url),user_book_partial(book_read,rating)',
  ...
})
```
https://docs.python.org/3/library/urllib.request.html#urllib-examples